utils.ts

```diff
@@ -0,0 +1,16 @@
+export const languages =
```

Comment:
This all exists in @components/magnolia/api no need to add this file or the methods.

Reply:
sure! 😊